src/settings.cpp

```diff
@@ -1542,7 +1557,11 @@ static void HandleSettingDescs(IniFile *ini, SettingDescProc *proc, SettingDescP
 	proc(ini, _win32_settings,   "win32", nullptr, only_startup);
 #endif /* _WIN32 */
 
-	proc(ini, _settings,         "patches",  &_settings_newgame, only_startup);
+	for (auto &table : _setting_tables) {
+		/* The name "patch" is a fallback, as every setting should sets its own group. */
```

Comment:
Shouldn't it be `patches` to be consistent with the next line?

Reply:
lol, yes, it should. No clue what I was smoking here, but clearly copy/pasting is too much to ask :D :D :D